### PR TITLE
How AMPs are cached (spanish)

### PR DESCRIPTION
**Pull request by SidVal**
From: https://github.com/ampproject/docs/commit/ab25305733ae45d9dc5578f0c9fe22b14f0bc602
Spanish translation

Fix: https://github.com/SidVal/docs/issues/48

### CODE REVIEW

File: content/docs/guides/pwa-amp/amp-as-pwa@es.md [hunk 1]
```diff
@@ -0,0 +1,119 @@
+---
+$title: Enable Progressive Web App features for your AMP pages
```
Comment (bpaduch) [inline content/docs/guides/pwa-amp/amp-as-pwa@es.md]: Why is this file included?  It's all in English?

File: content/docs/guides/pwa-amp/amp-in-pwa@es.md [hunk 2]
```diff
@@ -0,0 +1,132 @@
+---
+$title: Incruste y use AMP como fuente de datos
+$order: 1
+toc: true
+---
+[TOC]
+
```
Comment (bpaduch) [inline content/docs/guides/pwa-amp/amp-in-pwa@es.md]: Again, this file too is in English.

File: content/docs/guides/pwa-amp/amp-to-pwa@es.md [hunk 3]
```diff
@@ -0,0 +1,122 @@
+---
+$title: Preload your Progressive Web App from your AMP pages
```
Comment (bpaduch) [inline content/docs/guides/pwa-amp/amp-to-pwa@es.md]: Here too.

**Comment by SidVal** [comment]
No no, my mistake.
When I created this PR, that 3 files did not exists ! :roll_eyes:  hahahaha

**Comment by SidVal** [comment]
Now can be merged?
